shaunkendrone/Sacco-Frontend
# Ft register member

What does this PR do?
Elaborates how a user registers into the system.
Description of the Task to be completed?
To achieve registration, a user fills in the registration form with the required details and clicks on the Register button.
How should this be manually tested?
- Populate the registration form with the required details and click on the register button
- Try clicking on the register while leaving some fields empty. Once the register button is clicked messages in red appear on the empty fields showing that they are required.

Screenshots
![Screenshot 2023-07-18 142424](https://github.com/shaunkendrone/Sacco-Frontend/assets/98257312/33ba486f-77ed-47e2-ba15-1e18ce8b7a27)
